inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/deformable_convolution.hpp

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <string>
+#include <tuple>
+#include <vector>
+
+#include "ngraph_functions/builders.hpp"
+#include "ngraph_functions/utils/ngraph_helpers.hpp"
+#include "shared_test_classes/base/layer_test_utils.hpp"
+
+typedef std::tuple<InferenceEngine::SizeVector,  // Kernel size
+                   InferenceEngine::SizeVector,  // Strides
+                   std::vector<ptrdiff_t>,       // Pad begin
+                   std::vector<ptrdiff_t>,       // Pad end
+                   InferenceEngine::SizeVector,  // Dilation
+                   size_t,                       // Num out channel
+                   size_t,                       // Group
+                   size_t,                       // Deformable group
+                   ngraph::op::PadType           // Padding type
+                   >
+    deformableConvSpecificParams;
```

Comment:
I'm not sure the linter objects to this, but I think the style here in similar files is to have `>` and `deformableConvSpecificParams` on the same line without any preceding indentation.

Reply:
Hi Tim, Do you know which clang-format (and configurations) do OV developers use? It seems to me that different developers were using slightly different format config. The files under inference-engine and ngraph have different format as well.

Reply:
Thanks!

I unfortunately do not know which clang-format version / config the OV developers use. I am not aware of documentation on this. However, I am trying to make time to upstream changes from this repository into upstream OpenVINO, and it's possible that during this process I will be told about or referred to documentation on how OpenVINO style is managed.

All that said, the build process includes a cpplint step if you build in the OV repository directly (i.e., checkout this repository, make a build folder, run cmake, run make) rather than building via PlaidML, which will help correct some style issues. This can also catch a different collection of build errors (it's not an identical set of build tools or settings to what's in the PlaidML repo), and is part of what Frank did to find the problems he corrected in #143. I personally use the cmake settings `cmake -DENABLE_CLDNN=OFF -DENABLE_VPU=OFF -DENABLE_MYRIAD=OFF -DENABLE_FUNCTIONAL_TESTS=ON -DENABLE_TESTS=ON ..` when I do this, although it had been a while since I had done so (as evidenced by the necessity for #143).